gammapy/estimators/flux_point.py

```diff
@@ -302,7 +227,7 @@ def _convert_flux_columns(table, reference_model, sed_type):
         col_ref = table[sed_type]
         factor = fluxes[f"ref_{sed_type}"].to(col_ref.unit)
 
-        data = Table(fluxes)
+        data = Table(fluxes, meta=table.meta)
```

Comment:
I think that this implementation omits additional columns such as `ts` `counts` etc.

Reply:
Thanks, I'll check. It seems there are no unit tests yet that cover this case...

Reply:
The information is actual there, if one specifies the `sed_type="likelihood"` specifically:
```python3
fp = FluxPoints.read("flux-points.fits", sed_type="likelihood")
print(fp.counts)
```
Part of the problem is, that we wrote mixed sed types in v0.18.2 i.e. tables that contain all quantities like `norm` and `dnde` etc. But I guess we still want to keep quantities like `counts` for the other sed-types as well. So I'll check why those are not read from the table...

Reply:
Fixed